Perfect.

```
src/sentry/nodestore/migrations/0002_nodestore_no_dictfield.py:3:1: B318: The following __future__ are not allowed: absolute_import, division, print_function, unicode_literals
tests/sentry/nodestore/test_common.py:1:1: B318: The following __future__ are not allowed: absolute_import, division, print_function, unicode_literals
tests/sentry/api/endpoints/test_organization_sdk_updates.py:1:1: B318: The following __future__ are not allowed: absolute_import, division, print_function, unicode_literals
```

```
$ rg --hidden -S -g '!.git' __future__
src/sentry/nodestore/migrations/0002_nodestore_no_dictfield.py
3:from __future__ import unicode_literals
src/sentry/runner/commands/exec.py
70:    if "from __future__" in c:
75:            if line.startswith("from __future__"):
tests/sentry/nodestore/test_common.py
1:from __future__ import absolute_import
tests/sentry/api/endpoints/test_organization_sdk_updates.py
1:from __future__ import absolute_import
```

--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,8 @@ test:
 	py.test
 
 build: clean test lint fmt
-	python setup.py sdist bdist_wheel --universal > /dev/null
+	pip install -U wheel
+	python setup.py sdist bdist_wheel > /dev/null
 
 clean:
 	rm -rf __pycache__/ build/ dist/ *.egg-info/

--- a/setup.cfg
+++ b/setup.cfg
@@ -3,6 +3,7 @@ package_dir =
      = src
 
 [bdist_wheel]
+# We enforce python 3 specific minor version range in setup.py.
 python-tag = py3
 
 [flake8]

--- a/setup.cfg
+++ b/setup.cfg
@@ -3,7 +3,7 @@ package_dir =
      = src
 
 [bdist_wheel]
-python-tag = py36
+python-tag = py3
 
 [flake8]
 ignore = E203, E302, E501, W503, C901

--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,6 @@
 import sys
 
-if sys.version_info[:2] < (3, 6):
-    sys.exit("sentry-flake8 requires at least Python 3.6.")
+assert (3, 6, 0) <= sys.version_info < (3, 10, 0), "sentry-flake8 requires Python 3.6 - 3.9."
 
 from setuptools import setup  # noqa: e402
 
@@ -20,6 +19,7 @@ setup(
             "pytest==6.2.2",
         ]
     },
+    python_requires=">=3.6, <3.10",
     py_modules=["sentry_check"],
     entry_points={"flake8.extension": ["B = sentry_check:SentryCheck"]},
     classifiers=[

--- a/setup.py
+++ b/setup.py
@@ -1,8 +1,7 @@
 import sys
+from setuptools import setup
 
 assert (3, 6, 0) <= sys.version_info < (3, 10, 0), "sentry-flake8 requires Python 3.6 - 3.9."
-
-from setuptools import setup  # noqa: e402
 
 setup(
     name="sentry-flake8",

